Userland/Libraries/LibGUI/ComboBox.cpp

```diff
@@ -177,12 +177,12 @@ void ComboBox::navigate_relative(int delta)
 
 void ComboBox::selection_updated(ModelIndex const& index)
 {
-    if (index.is_valid())
+    if (index.is_valid()) {
         m_selected_index = index;
-    else
+        auto new_value = index.data().to_deprecated_string();
+        m_editor->set_text(new_value);
+    } else
```

Comment:
Changes LGTM, but since the `if` block now has braces, the `else` block also requires braces.

https://github.com/SerenityOS/serenity/blob/master/Documentation/CodingStyle.md#omission-of-curly-braces-from-statement-blocks
>  if any body of a connected if/else statement requires curly braces according to this rule, all of them do.